### PR TITLE
v0.26: Update telemetry table

### LINKED ISSUE #1419: v0.26: New telemetry datapoints
## Summary
The product team is interested in monitoring how many people use which instance options, so v0.26 will start sending that data to our analytics tools.

## Tasks
- [ ] update the table in the telemetry page with the new data points (check spec for full list)

## References
Spec: https://github.com/meilisearch/specifications/pull/107/files
Issue on core: https://github.com/meilisearch/MeiliSearch/issues/2088
Discussion: https://github.com/meilisearch/documentation/discussions/1365#discussioncomment-2052716_

### PR DESCRIPTION
**Pull request by dichotommy**
Fixes #1419

### CODE REVIEW

File: learn/what_is_meilisearch/telemetry.md [hunk 1]
```diff
@@ -103,14 +103,33 @@ This list is liable to change with every new version of Meilisearch. It's not be
...
+| `infos.ignore_missing_snapshot`         | `true` if `--ignore-missing-snapshot` is activated, otherwise `false`                       | true
+| `infos.ignore_snapshot_if_db_exists`    | `true` if `--ignore-snapshot-if-db-exists` is activated, otherwise `false`                  | true
+| `infos.http_addr`                       | `true` if `--http-addr`/`MEILI_HTTP_ADDR` is specified, otherwise `false`                   | true
+| `infos.max_index_size`                  | Value of `--max-index-size`/`MEILI_INDEX_SIZE` in bytes                                     | 336042103
+| `infos.max_task_db_size`                | Value of `--max-task-db-size`/`MEILI_MAX_TASK_DB_SIZE` in bytes                             | 336042103
+| `infos.http_payload_size_limit`         | Value of `--http-payload-size-limit`/`MEILI_HTTP_PAYLOAD_SIZE_LIMIT` in bytes               | 336042103
+| `infos.enable_autobatching`             | `true` if `--enable-autobatching` is activated, otherwise `false`                           | true
+| `infos.max_batch_size`                  | Value of `--max-batch-size` in integer, otherwise `null`                                    | 1000
```
Comment (maryamsulemani97) [inline learn/what_is_meilisearch/telemetry.md:123]: Is it me, or does "in integer" sound weird?

**Review by maryamsulemani97 — COMMENTED** [review]
Looks good! 🍪
Not relevant to this PR, but do we want to add how often this data is collected?

**Comment by dichotommy** [comment]
> Do we want to add how often this data is collected?

It's explained just above the table!

File: learn/what_is_meilisearch/telemetry.md (hunk 1 above)
Comment (dichotommy) [reply learn/what_is_meilisearch/telemetry.md:123]: Not just you, it totally sounds weird

**Review by guimachiavelli — APPROVED** [review]
bors merge

**Pull request merged**